src/main/java/com/auth0/json/mgmt/logstreams/LogStream.java

```diff
@@ -0,0 +1,114 @@
+package com.auth0.json.mgmt.logstreams;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.Map;
+
+/**
+ * Represents an Auth0 Log Stream object. Related to the {@linkplain com.auth0.client.mgmt.LogStreamsEntity}
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LogStream {
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("name")
+    private String name;
+    @JsonProperty("type")
+    private String type;
+    @JsonProperty("status")
+    private String status;
+    @JsonProperty("sink")
+    private Map<String, Object> sink;
```

Comment:
The `sink` schema of a Log Stream varies depending on the `type`. Instead of going down the path of implementing polymorphic serializing/deserializing with Jackson, using a map of `<String, Object>`. Currently only string values are used, but future additional types may support other sink attribute types (specifically booleans).